### PR TITLE
Add countyl010 file

### PR DESCRIPTION
**Pull request by michaelavs**

### CODE REVIEW

**Comment by anissa111** [comment]
Shouldn't the supporting files also be added to registry.txt?

**Comment by anissa111** [comment]
Adding @erogluorhan to make sure I'm not mistaken about needing to add the supporting files to the registry, too

**Review by pilotchute — APPROVED** [review]

**Comment by erogluorhan** [comment]
> Adding @erogluorhan to make sure I'm not mistaken about needing to add the supporting files to the registry, too

I agree with @anissa111 . A separate registry entry per each file added should be made, which is the supporting files (i.e. .dbf, .prj, .shx) in this case.

File: registry.txt [hunk 1]
```diff
@@ -309,6 +309,7 @@ netcdf_files/wrfout_d03_2012-04-22_23_00_00_Z.nc 5093f2fbea7eae5b6d45869b925dd19
 netcdf_files/wrfout_d03_2012-04-22_23_00_00_subset.nc 6e05226406066da86773c76023eafb727e9540577c90c0c7f95e01eaf19c7208
 netcdf_files/xieArkin-T42.nc 3bac3b85943cc1f07d8a30295d2f880087945f38d11ca64d61e1c0434c90377f
 netcdf_files/za_PHC2_T_S_gx1v3.nc 847eee673e3f2afb4793387f4318243f7be7929eca39acc3bab91f4083348986
+shape_files/countyl010g.shp e1f68233489c7abaaf7a8d7c8dee5583d0c65c5397f26e78f26bf32c764cfd53
```
Comment (erogluorhan) [inline registry.txt:314]: Make sure to also add registry entries for the other files added (.prj, .shx, .dbf)

**Review by erogluorhan — CHANGES_REQUESTED** [review]
See inline comment

File: registry.txt (hunk 1 above)
Comment (anissa111) [reply registry.txt:314]: Done!

**Review by erogluorhan — APPROVED** [review]

**Review by anissa111 — APPROVED** [review]

**Pull request merged**